core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -186,8 +188,13 @@
     private Long shmSize;
 
     // Maintain order in which entries are added, as earlier target location may be a prefix of a later location.
+    @Deprecated
     private Map<MountableFile, String> copyToFileContainerPathMap = new LinkedHashMap<>();
 
+    // Maintain order in which entries are added, as earlier target location may be a prefix of a later location.
+    @Setter(AccessLevel.NONE)
```

Comment:
Let's also add `@Getter(AccessLevel.NONE)`

Reply:
Ah, okay. Then a tiny nit: `@Getter(AccessLevel.MODULE)` would do 👍

Reply:
I wanted to get the @visiblefortesting annotation on the getter - but it's your call.

Reply:
done